source/system/math.ooc

```diff
@@ -240,6 +240,15 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	sineInterpolation: func -> Float {
+		result := 0.0f
+		absoluteValue := this absolute
+		if (absoluteValue > 1.0f)
+			result = 1.0f
+		else
+			result = (sin((absoluteValue - 0.5f) * pi) + 1.0f) / 2.0f
+		result
```

Comment:
The above five lines can be written simply as
`absoluteValue > 1.f ? 1.f : (sin((absoluteValue - 0.5f) * pi) + 1.0f) / 2.0f`